packages/ui5-application-inquirer/src/i18n.ts

```diff
@@ -1,7 +1,8 @@
 import type { TOptions } from 'i18next';
 import i18next from 'i18next';
 import translations from './translations/ui5-application-inquirer.i18n.json';
-import { addi18nResourceBundle as addInquirerCommoni18nResourceBundle } from '@sap-ux/inquirer-common';
+import { addi18nResourceBundle as addi18nResourceBundleInquirerCommon } from '@sap-ux/inquirer-common';
```

Comment:
How is a developer going to know this?

Reply:
You wont, we need a TBI to fix all i18n text loading generally as this is a suboptimal solution!

Reply:
Thanks.